src/endpoints/getPlayerByName.ts

```diff
@@ -0,0 +1,33 @@
+import { HLTVConfig } from '../config'
+import axios from 'axios'
+import { ISOCodesToLong } from '../utils/parsing'
+import { SearchPlayer } from 'models/SearchPlayer'
+
+export const getPlayerByName = (config: HLTVConfig) => async ({
+  name
+}: {
+  name: string
+}): Promise<SearchPlayer> => {
+  return axios.get(`${config.hltvUrl}/search?term=${name}`)
```

Comment:
Why don't you use `config.loadPage` here? I don't like adding new dependencies and axios is definitely something I wouldn't want in this library.

Reply:
Beause the response is not a html page but json. I could use fetch insteed or is ther a option to edit json with the `config.loadPage` function ?

Reply:
loadPage returns a string with the response from the request, but I think you should be able to `JSON.parse` it into an object.